IanGRap/Squish
# Long and Tall Block Shapes

Contributors: Coulter, Alfredo

Goal: Create tall block (1x3) and long block (3x1).

Files Changed:
- GetFormationLength.cs: Simple script to pass the shape's width value to the spawner to avoid spawning outside or on top of the walls.
- Spawner.cs: Added tall and long block prefabs and formations to the spawn formation list. Also adjusted spawn position shift calculations to account for width values passed by GetFormationLength.cs.
- BlockClearer.cs: Added _overlapCount variable to keep track of how filled the bottom row is (instead of using the list count).
- SquishDetector.cs: Removed unneeded bool that was messing up all block squishing in the downward direction.

How to test: 
- Go into "First Release 1" scene
- Run the game
- Interact with the shapes, observe their interactions with each other (and with special blocks)

Notes: There's a camera manager error popping up occasionally, not our fault. Shout out to Austin for fixing his fuck-up.
